backend/inspirehep/records/api/base.py

```diff
@@ -306,27 +334,60 @@ def commit(self, *args, **kwargs):
         This method does nothing, instead all the work is done in ``update``.
         """
 
+    def redirect_pid(self, pid_type, pid_value):
+        try:
+            old_pid = PersistentIdentifier.get(pid_type, pid_value)
+        except PIDDoesNotExistError:
+            LOGGER.warning(
```

Comment:
why would this scenario happen? it should probably be a more serious thing than just a warning.

Reply:
This will happen now for every record which redirects a record we do not have in our DB. As you said some time before we have records which are redirecting records which were deleted and are not migrated.
```
Micha Moskovic10:56
As I said, we have deleted records that don't migrate
```

Reply:
:+1: I hadn't thought of this